sdk/src/main/java/com/horizen/backwardtransfer/BackwardTransferFunctions.java

```diff
@@ -0,0 +1,37 @@
+package com.horizen.backwardtransfer;
+
+import com.horizen.box.WithdrawalRequestBox;
+
+import java.util.EnumMap;
+import java.util.List;
+
+public interface BackwardTransferFunctions {
```

Comment:
I have doubts about usage of `...Functions`, `Loader` and structure of vrf, schnorr, zkproofs classes. We need to clarify asap what is going to be a final version and exact usage of rust/jni library.

Reply:
agree